python/pyspark/shell.py

```diff
@@ -63,10 +63,32 @@
 """
     % sc.version
 )
-print(
-    "Using Python version %s (%s, %s)"
-    % (platform.python_version(), platform.python_build()[0], platform.python_build()[1])
-)
+
+
+def _create_version_string() -> str:
```

Comment:
Hm, I think it's a bit odd to print these out in the Spark shell. Do you have any such reference like Jupyter or other shells? I would expect users to just print it out by themselves e.g., `pip list`

Reply:
I could see python version, but a bunch of other dependencies seems noisy